### PR TITLE
UniswapV3LiquidatorFunder config for the bots

### PR DESCRIPTION
**Pull request by coolnft**
## Type of change

- [ ] Docs change / dependency upgrade
- [ ] Configuration / tooling changes
- [ ] Refactoring
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires changes in customer code

## Related Issue(s)

Fixes

### CODE REVIEW

File: packages/chains/src/arbitrum/params.ts [hunk 1]
```diff
@@ -10,6 +10,7 @@ const specificParams: ChainParams = {
     chainIdHex: "0xA4B1",
     name: "Arbitrum One",
     shortName: "Arbitrum",
+    uniswapV3Fee: 3000,
```
Comment (vminkov) [inline packages/chains/src/arbitrum/params.ts:13]: should there be any configuration of the fee level for different assets/pairs?

File: packages/types/src/chain.ts [hunk 2]
```diff
@@ -58,6 +59,7 @@ export type ChainAddresses = {
   W_TOKEN_USD_CHAINLINK_PRICE_FEED: string;
   UNISWAP_V2_ROUTER: string;
   UNISWAP_V2_FACTORY: string;
+  UNISWAP_V3_ROUTER?: string;
```
Comment (carlomazzaferro) [inline packages/types/src/chain.ts:62]: I suggest we use something like this: https://github.com/Midas-Protocol/monorepo/blob/feature/security/packages/types/src/chain.ts

Comment (carlomazzaferro) [reply packages/types/src/chain.ts:62]: Because each chain will also likely need as specific PAIR_INIT_HASH and quoter addresses

Comment (coolnft) [reply packages/types/src/chain.ts:62]: we don't need `pair_init_hash` for v3 and we can get `quoter` address from deployments.

File: packages/chains/src/arbitrum/params.ts (hunk 1 above)
Comment (coolnft) [reply packages/chains/src/arbitrum/params.ts]: @vminkov @carlomazzaferro please check this update.

Comment (carlomazzaferro) [reply packages/chains/src/arbitrum/params.ts]: Looks good now

**Review by vminkov — APPROVED** [review]

**Pull request merged**